web/modules/custom/joinup_search/config/install/search_api.index.published.yml

```diff
@@ -415,13 +416,41 @@ processor_settings:
       - field_isr_description
       - field_ar_abstract
       - field_is_description
+      - is_field_ci_name
+      - is_field_owner_name
       - field_user_first_name
       - field_user_family_name
       - field_user_organisation
+    spaces: ''
+    overlap_cjk: 1
+    minimum_word_size: '3'
     weights:
-      preprocess_index: -50
-      preprocess_query: -50
-  rendered_item: {  }
+      preprocess_index: -49
+      preprocess_query: -49
+  type_boost:
+    boosts:
+      'entity:node':
+        datasource_boost: !!float 1
+        bundle_boosts:
+          custom_page: !!float 5
+          discussion: !!float 2
+          document: !!float 0
+          event: !!float 3
+          news: !!float 4
+          newsletter: !!float 0
+          video: !!float 0
+      'entity:rdf_entity':
+        datasource_boost: !!float 1
+        bundle_boosts:
+          asset_distribution: !!float 0
+          asset_release: !!float 0
+          collection: !!float 7
```

Comment:
Same as for `node:news`. This can go with 21.0. And `datasource_boost` should be 0.0 according to the ticket.